amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -161,17 +161,28 @@ enum ColonyActionType {
   GENERIC
   WRONG_COLONY
   PAYMENT
+  PAYMENT_MOTION
```

Comment:
From my initial work on the block ingestor, I get the sense we'll be able to share a model for actions and motions. 

@jakubcolony Let me know if you see a problem with this. 

Also, it may be that we can do away with having separate types for actions and motions and just store an isMotion flag on the model. But just speculating at the moment, it may become clearer as work carries on that having distinct types is desirable.